Reminder-App/Reminder-App/ReminderListDataSource.swift

```diff
@@ -9,23 +9,26 @@ import UIKit
 
 class ReminderListDataSource: NSObject {
     
-    // MARK:- Static Properties
+    // MARK: Static Properties
     static let reminderListCellIdentifier = "ReminderListCell"
     
-    // MARK:- Private Properties
+    // MARK: Private Properties
     private lazy var dateFormatter = RelativeDateTimeFormatter()
 }
 
-// MARK:- Data Source Methods
+// MARK: Data Source Methods
 extension ReminderListDataSource: UITableViewDataSource {
     
     func tableView(_ tableView: UITableView, numberOfRowsInSection section: Int) -> Int {
+        
         return Reminder.testData.count
     }
+    
     func tableView(_ tableView: UITableView, cellForRowAt indexPath: IndexPath) -> UITableViewCell {
+        
         guard let cell = tableView.dequeueReusableCell(withIdentifier: Self.reminderListCellIdentifier, for: indexPath)
                 as? ReminderListCell else {
-            fatalError("Unable to dequeue ReminderCell")
+            return
```

Comment:
that `return` would not work here. as this function requires a `UITableViewCell` as a return type. Make sure your code does not have any errors before creating a PR